cache/redis/redis.go

```diff
@@ -19,24 +21,35 @@ const (
 	ConfigKeyDB       = "db"
 	ConfigKeyMaxZoom  = "max_zoom"
 	ConfigKeyTTL      = "ttl"
+	ConfigKeySSL      = "ssl"
 )
 
-func init() {
-	cache.Register(CacheType, New)
-}
+var (
+	// default values
+	defaultNetwork  = "tcp"
+	defaultAddress  = "127.0.0.1:6379"
+	defaultPassword = ""
+	defaultDB       = 0
+	defaultMaxZoom  = uint(tegola.MaxZ)
+	defaultTTL      = 0
+	defaultSSL      = false
+)
 
-func New(config dict.Dicter) (rcache cache.Interface, err error) {
+// ErrHostMissing is raised when Redis Addr is missing Host
+type ErrHostMissing struct {
+	msg string
+}
 
-	// default values
-	defaultNetwork := "tcp"
-	defaultAddress := "127.0.0.1:6379"
-	defaultPassword := ""
-	defaultDB := 0
-	defaultMaxZoom := uint(tegola.MaxZ)
-	defaultTTL := 0
+func (error *ErrHostMissing) Error() string {
+	return error.msg
+}
```

Comment:
Can we move this to a file called, `cache/redis/errors.go`. This is just to keep the convention of having all our errors for a package declared in errors.go.